user-org-actor/src/main/java/org/sunbird/user/dao/UserOSDaoImpl.java

```diff
@@ -61,4 +64,23 @@ public Response createUser(Map<String, Object> user) throws BaseException {
             throw new ProjectCommonException.ServerError(IResponseMessage.INTERNAL_ERROR, localizer.getMessage(IResponseMessage.INTERNAL_ERROR, null), ResponseCode.SERVER_ERROR.getCode());
         }
     }
+
+    /**
+     * this method is used to read user record from OS.
+     * @param inputNode
+     * @return response
+     * @throws BaseException
+     */
+    @Override
+    public Response readUser(JsonNode inputNode) throws BaseException {
+        Response response = new Response();
+        try {
+            JsonNode responseNode = registryHelper.readEntity(inputNode,"");
```

Comment:
use overide method for readEntity with one and two argument .

Reply:
This change should be at user-registry, i hope here it is not necessary.